### PR TITLE
Add native date picker for send later

### LINKED ISSUE #8682: Send later should use native datetime picker
### Steps to reproduce

1. Open the app
2. Configure an account
3. Start composing a new message
4. Select *Send later*
5. Use the time picker

### Expected behavior

accessible, native picker

the picker is also very flickery and I don't see all contents

Bildschirmfoto vom 2023-08-08 12-26-01
Bildschirmfoto vom 2023-08-08 12-23-37

### Actual behavior

custom picker

### Mail app version

3

### Mailserver or service

_No response_

### Operating system

_No response_

### PHP engine version

None

### Web server

None

### Database

None

### Additional info

https://github.com/nextcloud-libraries/nextcloud-vue/issues/1666

### PR DESCRIPTION
**Pull request by GretaD**
fixes #8682 

before
Screenshot from 2023-08-08 14-12-33

after
Screenshot from 2023-08-08 14-42-20

### CODE REVIEW

**Comment by GretaD** [comment]
i also removed the selection of earlier dates

File: src/components/Composer.vue [hunk 1]
```diff
@@ -379,14 +379,15 @@
...
+							:min="dateToday"
 							type="datetime-local"
 							:first-day-of-week="firstDayDatetimePicker"
 							:use12h="showAmPm"
 							:formatter="formatter"
 							:format="'YYYY-MM-DD HH:mm'"
 							icon=""
 							:minute-step="5"
-							:show-second="false"
 							:disabled-date="disabledDatetimepickerDate"
```
Comment (GretaD) [inline src/components/Composer.vue:391]: i dont know what this does and if we need it

Comment (ChristophWurst) [reply src/components/Composer.vue:391]: I think you can drop disabledDatetimepickerDate and disabledDatetimepickerTime

**Review by ChristophWurst — APPROVED** [review]
So much better

File: src/components/Composer.vue (hunk 1 above)
Comment (GretaD) [reply src/components/Composer.vue:391]: ah, ok, then its not working so dropping it off

**Pull request merged**